File: 042-左旋转字符串/042-LeftRotateString.cpp
class Solution {
public:
    string LeftRotateString(string str, int n) {
        string ans(str.size(),'0');
        for(int i =0;i<str.size();i++){
            ans[i]=str[(i+n)%str.size()];
        }
        return ans;
    }
};

File: 042-左旋转字符串/042-LeftRotateString.py
# -*- coding:utf-8 -*-
class Solution:
    def LeftRotateString(self, s, n):
        # write code here
        #return s[n:]+s[:n]
        ans=[0]*len(s)
        for i in range(len(s)):
            ans[i]=s[(i+n)%len(s)]
        return ''.join(ans)
